### ISSUE TITLE
Test on webcam

### ISSUE THREAD

**Issue by DaddyWesker** [DenisRang/Combined-CNN-RNN-for-emotion-recognition#1]
Hello. THanks for your code.

If i got it right, to test your code on webcam or some sort of video, i need a batch of 100 features of consequent frames and send it to 
```
 x = prepare_sequence_for_rnn(sequence)
 prediction = rnn_model.predict(x)
```
?
Also, i'll be glad to know if i need to pre-detect face on the image? And what size image should be? THanks

P.S. Well, i guess your model just told me about size 
`Input 0 is incompatible with layer model: expected shape=(None, 96, 96, 1), found shape=(None, 480, 640, 3)`
But question about face detect is still there.

**Comment by DaddyWesker** [comment]
Well, i've looked through your paper and found that i do need to pre-detect face. But now i've got another question. Your rnn outputs arousal and valence. But how to (if possible) transform it to simple emotion labels? I mean, i want just to evaluate your model as a simple emotion classifier, but probably i'm doing something wrong? Here is how it is done right now. ANy help will be appreciated.

```
import os

import numpy as np
import pandas as pd
from tensorflow.keras.models import load_model
from tqdm.notebook import tqdm

from src import metrics
from src.config import PREDICTIONS, RNN_WINDOW_SIZE, \
    SAVED_CNN_EXTRACTOR_MODEL, SAVED_RNN_MODEL
from src.data import DataSet
from src.extractor import Extractor
from src.utils import VideoHelper
import cv2

video_capture = cv2.VideoCapture(0)

face_detector = cv2.CascadeClassifier("/home/daddywesker/UsableModels/haarcascade_frontalface_default.xml")
cnn_extractor_model = Extractor(SAVED_CNN_EXTRACTOR_MODEL)
rnn_model = load_model(SAVED_RNN_MODEL, custom_objects={'ccc_loss': metrics.ccc_loss,
                                                        'rmse': metrics.rmse,
                                                        'rmse_v': metrics.rmse_v,
                                                        'rmse_a': metrics.rmse_a,
                                                        'cc_v': metrics.cc_v,
                                                        'cc_a': metrics.cc_a,
                                                        'ccc_v': metrics.ccc_v,
                                                        'ccc_a': metrics.ccc_a})


sequence = []
while True:
    _, frame = video_capture.read()
    frame = cv2.cvtColor(frame, cv2.COLOR_RGB2GRAY)
    faces = face_detector.detectMultiScale(
        frame,
        scaleFactor=1.1,
        minNeighbors=5,
        flags=cv2.CASCADE_SCALE_IMAGE,
        minSize=(50, 50))
    x, y, width, height = faces[0]
    x1, x2, y1, y2 = (x - 10, x - 10 + width, y - 10, y - 10 + height)
    frame = frame[x1:x2, y1:y2]
    frame = cv2.resize(frame, (96, 96), interpolation=cv2.INTER_CUBIC)

    feature_vector = cnn_extractor_model.extract_image(frame)
    sequence.append(feature_vector)
    if len(sequence) < RNN_WINDOW_SIZE:
        continue
    x = np.expand_dims(np.asarray(sequence), 0)
    prediction = rnn_model.predict(x)
    sequence.pop(0)
```

**Comment by armandem21** [comment]
can you run the code for default Aff-Wild2 dataset?
